src/lib.rs

```diff
@@ -615,8 +680,8 @@ pub enum PolicySource {
 #[derive(Clone, Debug, Eq, PartialEq)]
 #[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
 pub struct Directive {
-    name: String,
-    value: Vec<String>,
+    pub name: String,
```

Comment:
Needs to be public so that we can pass on the directive name into the CSP violations reporter: https://github.com/servo/servo/blob/4148b0dc9466563bdb88df8b472371717b1ea272/components/script/dom/globalscope.rs#L3328

Reply:
Yes, this is fine, too.